frontend/public/components/masthead.jsx

```diff
@@ -6,7 +6,6 @@ import { MastheadToolbar } from './masthead-toolbar';
 import { history } from './utils';
 import okdLogoImg from '../imgs/okd-logo.svg';
 import openshiftLogoImg from '../imgs/openshift-logo.svg';
-import ocpLogoImg from '../imgs/openshift-platform-logo.svg';
```

Comment:
Should also delete the logo file itself since it is not longer in use.

Reply:
updated with deleted file